mathesar/rpc/forms.py

```diff
@@ -80,6 +80,7 @@ class FormInfo(TypedDict):
         description: The description of the form.
         version: The version of the form.
         database_id: The Django id of the database containing the Form.
+        server_id: The Django id of the server cantaining the database.
```

Comment:
There's a typo. (`cantaining`). But, I actually think this should be invisible to the client. I.e., there's no reason for the front end to submit, or get info about, the server ID for the form. Just handle it in the `mathesar.utils.forms.create_form` function. Especially since we eventually want to extract the Database Role Map from `UserDatabaseRoleMap`. We don't want to change the API at that point.